test/data/two_pop/ld_reg.cf_expected

```diff
@@ -0,0 +1 @@
+0.9999999999999986	1.000000000000004	1.000000000000004	0.9999999999999993
```

Comment:
Do we want new lines here?

Reply:
I had thought about doing that, but this is exactly how numpy generates the file (I took the output of a good run), so I'm hesitant to change it, but it would probably be fine if it's something that will be bothersome,

Reply:
Makes sense!